src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -707,7 +707,9 @@ def setUp(self):
         # Access points to use by default when starting servers and agents
         #  - for 1 or 2 servers use 1 access point
         #  - for 3 or more servers use 3 access points
-        access_points_qty = 1 if len(self.hostlist_servers) < 3 else 3
+        default_access_points_qty = 1 if len(self.hostlist_servers) < 3 else 3
+        access_points_qty = self.params.get(
+            "access_points_qty", "/run/setup/*", default_access_points_qty)
```

Comment:
It may be a good idea to add a check for the user input:
```suggestion
            "access_points_qty", "/run/setup/*", default_access_points_qty)
        if access_points_qty < 1 or access_points_qty > len(self.hostlist_servers):
            self.fail("Invalid access points node quantity")
```